src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/component/AuthenticatedClient.kt

```diff
@@ -188,6 +192,19 @@ class AuthenticatedClient(
     sendPostRequest("/recalls/$recallId/stop", StopRecallRequest(stopReason))
       .expectStatus().isEqualTo(OK)
 
+  fun startPhase(recallId: RecallId, phase: Phase, expectedStatus: HttpStatus = CREATED) =
```

Comment:
We don't need to change anything right now but FYI I do wonder why we have e.g. two methods for `startPhase` where one does not call the other and hence they duplicate the URL.  Then the `endPhase` seems like another model again - where e.g. the expectBody is explicit not a parameter to a generic `patchRequest` method? 🤷‍♂️

Reply:
the 2 methods is something we have in lots of places. Where we dont return an object (e.g. it errors) we need to have 1 method, and then the 2nd translates the response into the class.
As for the patch, we dont return anything on the other patch calls, so needed to add it here. 

Possibly we can look at rewriting all the AuthClient at some point, but there always seems to be loads of different permutations. maybe a smell that we aren't consistent in our responses/behaviours